pmm/pmm2-ovf-staging-start.groovy

```diff
@@ -105,5 +139,83 @@ pipeline {
                 }
             }
         }
+        stage('Enable Testing Repo') {
+            when {
+                expression { env.ENABLE_TESTING_REPO == "yes" && env.ENABLE_EXPERIMENTAL_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME){
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
+                                sudo yum update -y percona-release || true
+                                sudo sed -i'' -e 's^/release/^/testing/^' /etc/yum.repos.d/pmm2-server.repo
+                                sudo percona-release enable percona testing
+                                sudo yum clean all
+                            '
+                        """
+                    }
+                }
+            }
+        }
+        stage('Enable Experimental Repo') {
+            when {
+                expression { env.ENABLE_EXPERIMENTAL_REPO == "yes" && env.ENABLE_TESTING_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME){
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
+                                sudo yum update -y percona-release || true
+                                sudo sed -i'' -e 's^/release/^/experimental/^' /etc/yum.repos.d/pmm2-server.repo
+                                sudo percona-release enable percona experimental
+                                sudo yum clean all
+                            '
+                        """
+                    }
+                }
+            }
+        }
+        stage('Enable Release Repo') {
+            when {
+                expression { env.ENABLE_EXPERIMENTAL_REPO == "no" && env.ENABLE_TESTING_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME) {
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
+                                sudo yum update -y percona-release || true
+                                sudo yum clean all
```

Comment:
Do we need this block? We don't do here anything.

Also, maybe it makes sense to use a function instead of this block three times:
```
def enableRepo(REPO string) {
                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
                        sh """
                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
                                sudo yum update -y percona-release || true
                                sudo sed -i'' -e 's^/release/^/$REPO/^' /etc/yum.repos.d/pmm2-server.repo
                                sudo percona-release enable percona $REPO
                                sudo yum clean all
}
```

Reply:
This has been done